libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -273,6 +273,9 @@
         OPENED_MEDIA_LIBRARY,
         OPENED_BLOG_SETTINGS,
         OPENED_ACCOUNT_SETTINGS,
+        ACCOUNT_SETTINGS_CHANGE_USERNAME_SUCCEEDED,
```

Comment:
Great job considering track events! However, you also need to add names for the events here **https://github.com/wordpress-mobile/WordPress-Android/blob/04cf520ef21257b9e75a94c4f47941235fd0ad47/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L598**.

We are showing a warning in the LogCat when the track event name isn't found: `There is NO match for the event`. Thinking about it now, we might want to consider crashing the app in debug builds as it's pretty easy to make such mistake - especially for new comers.

Reply:
Could you please create a ticket, so we don't forget about it! Feel free to assign it to me. Thanks!

Reply:
Perfect, thank you !

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10436/commits/4a1a2613fdb7d42a3851d3ab80163821c9ece92b